data.sql

```diff
@@ -1,5 +1,2 @@
 /* Populate database with sample data. */
-
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (0, 'Agumon', '2020-02-3', 0, TRUE, 10.23),(1, 'Gabumon', '2018-11-18', 2, TRUE, 8),(2, 'Pikachu', '2021-01-7', 1, FALSE, 15.04),(3, 'Devimon', '2017-5-12', 5, TRUE, 11);
```

Comment:
kindly note that the data to be inserted for Gabumon is ` INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Gabumon', '2018/11/15', '2', 'true', '8.0');
``
please fill in the correct value for date_of_birth for Gabumon 😄

Reply:
@usorfaitheloho how is it for Gabumon is as you and for the other values you are not expecting me to make changes when they are clearly the same formart